docs/dapps/api-aliases.json

```diff
@@ -2,93 +2,125 @@
     "current": [
         {
             "from": "https://wallet.mainnet.alephium.org",
-            "to": "https://wallet-v15.mainnet.alephium.org"
+            "to": "https://wallet-v16.mainnet.alephium.org"
         },
         {
             "from": "https://wallet.testnet.alephium.org",
-            "to": "https://wallet-v15.testnet.alephium.org"
+            "to": "https://wallet-v16.testnet.alephium.org"
         },
         {
             "from": "https://backend.mainnet.alephium.org",
-            "to": "https://backend-v111.mainnet.alephium.org"
+            "to": "https://backend-v112.mainnet.alephium.org"
         },
         {
             "from": "https://backend.testnet.alephium.org",
-            "to": "https://backend-v111.testnet.alephium.org"
+            "to": "https://backend-v112.testnet.alephium.org"
         },
         {
             "from": "https://explorer.mainnet.alephium.org",
-            "to": "https://explorer-v111.mainnet.alephium.org"
+            "to": "https://explorer-v112.mainnet.alephium.org"
         },
         {
             "from": "https://explorer.testnet.alephium.org",
-            "to": "https://explorer-v111.testnet.alephium.org"
+            "to": "https://explorer-v112.testnet.alephium.org"
+        },
+        {
+            "from": "https://explorer.alephium.org",
+            "to": "https://explorer-v112.mainnet.alephium.org"
+        },
+        {
+            "from": "https://testnet.alephium.org",
```

Comment:
@killerwhile I think this endpoint should be deprecated (redirected) in favor of `explorer.testnet.alephium.org`. The latter one should be the publicly visible endpoint. 
@mvaivre @killerwhile WDYT?

Reply:
Yes, this would make sense.